htdocs/mri/jiv/get_file.php

```diff
@@ -92,6 +94,12 @@
     exit(4);
 }
 
+// If $File contains "DCM_", prefix automatically inserted by the
+// LORIS-MRI pipeline, identify it as $FileExt: "DICOMTAR"
+if (strpos($File, "DCM_") ) {
```

Comment:
missing `=== 0`. The comment should also point out that it's a hack, since there's no such thing as a DICOMTAR extension.

Reply:
@driusan 
you can not assume the `DCM` is at the beginning of the filename. It used to be the case, but not anymore (now the file stored within a year directory, followed by the filename which then starts with `DCM_`), so in the tarchive table, a typical ArchiveLocation is:
`2016/DCM_2016-09-13_ImagingUpload-9-50-abCD77.tar`

Reply:
In that case, it still needs `!== false` to differentiate from finding at the 0 index vs not finding in the string at all.

Reply:
thanks, done!